main.rb

```diff
@@ -0,0 +1,45 @@
+require './app'
+def main
+  books = []
+  rental_data = []
+  people = []
+
+  options(books, people, rental_data)
+end
+
+def display_all
+  puts 'Welcome to the school library'
+  puts ' Please choose a task  basing on the number '
+  puts "   1:  Show all books.
+           2:  Show all people.
+           3: Create a person
+           4: Create a rental
+           5: Create a book
+           6: List rented books to a person by ID
+           7: Exit"
```

Comment:
- [ ] [OPTIONAL] In my opinion it will be nice and logical if you replace the `4` and `5` lines and pay attention to the layout structure like [provided by Microverse](https://www.youtube.com/watch?v=vkkgrhD6aXQ) 👍🏼 

![image](https://user-images.githubusercontent.com/63915024/210782195-20de6e91-f512-4d96-80e0-7f11a5f467aa.png)